### PR TITLE
Fix SingleLineMethodDeclarationWhenPossibleSniff

### PR DESCRIPTION
**Pull request by grongor**

### CODE REVIEW

File: src/Cdn77/Sniffs/Classes/SingleLineMethodDeclarationWhenPossibleSniff.php [hunk 1]
```diff
@@ -51,7 +51,9 @@ public function process(File $file, $pointer) : void
         $tokens = $file->getTokens();
 
         $lineStartPtr = $file->findFirstOnLine(T_WHITESPACE, $pointer);
-        assert(is_int($lineStartPtr));
+        if ($lineStartPtr === false) {
```
Comment (simPod) [inline src/Cdn77/Sniffs/Classes/SingleLineMethodDeclarationWhenPossibleSniff.php]: What's ptr?

Comment (grongor) [reply src/Cdn77/Sniffs/Classes/SingleLineMethodDeclarationWhenPossibleSniff.php]: pointer, standard in cs sniffs

Comment (simPod) [reply src/Cdn77/Sniffs/Classes/SingleLineMethodDeclarationWhenPossibleSniff.php]: Can we rather make it "pointer" so we drop some WTFs from code? The +4 symbol cost is not significant :)

Comment (grongor) [reply src/Cdn77/Sniffs/Classes/SingleLineMethodDeclarationWhenPossibleSniff.php]: done

**Pull request merged**